packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -133,13 +133,15 @@ export const startPSM = async (
 
   const marshaller = await E(board).getPublishingMarshaller();
 
+  const instanceKey = `psm-${Stable.symbol}-${keyword}`;
```

Comment:
Again, If hyphens are used to separate components of the vat identifier, it would be good to use a different separator within sub-parts.

Reply:
I'm changing the others from `-` to `.` as requested, but this `instanceKey` is part of an existing API for finding PSM instances in `agoricNames.instance`.

_darn it; #7229 is supposed to document he `agoricNames` API, but it only shows `agoricNames.instance.psm`._

example client:

https://github.com/Agoric/dapp-psm/blob/afa0e2c51e7629985a487ca860f66be409f930ec/src/utils/updates.ts#L60-L61